pulsar-testclient/pom.xml

```diff
@@ -94,6 +94,13 @@
 			<groupId>org.hdrhistogram</groupId>
 			<artifactId>HdrHistogram</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.json</groupId>
+			<artifactId>json</artifactId>
+			<version>20201115</version>
+			<scope>test</scope>
+		</dependency>
```

Comment:
The license of this library is `The Software shall be used for Good, not Evil.` and it's not really compatible with Apache license. We generally use Jackson and it's already in class path.

Reply:
Thanks for the feedback, I will take a look at updating.

Reply:
Move completed to jackson.